Data/Vector/Generic.hs

```diff
@@ -99,7 +99,7 @@ module Data.Vector.Generic (
   -- ** Filtering
   filter, ifilter, uniq,
   mapMaybe, imapMaybe,
-  filterM,
+  filterM, mapMaybeM, imapMaybeM,
```

Comment:
Not terribly important, but for consistency sake. All other modules export `mapMaybeM` and `imapMaybeM` before `filterM` and after `mapMaybeM`, `imapMaybe`.

```suggestion
  mapMaybeM, imapMaybeM,
  filterM,
```

Reply:
Data.Vector wasn't consistent as well. I arranged everything in the same way. I also put `filterM` after `filter` & `ifilter`. I think it makes sense to keep all filter variants together.